manifests/b/btpf/Alexandria/0.13.1/btpf.Alexandria.installer.yaml

```diff
@@ -0,0 +1,28 @@
+# Created with komac v2.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: btpf.Alexandria
+PackageVersion: 0.13.1
+InstallerType: portable
+InstallModes:
+- interactive
+UpgradeBehavior: install
+FileExtensions:
+- azw
+- azw3
+- cb7
+- cbr
+- cbt
+- cbz
+- epub
+- epub3
+- fb2
+- fb2.zip
+- txt
+ReleaseDate: 2024-10-06
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/btpf/Alexandria/releases/download/v0.13.1/Alexandria_0.13.1_x64-portable.exe

```

Comment:
They are also providing `*setup.exe` and `.msi`. Should I prefer it rather than portable?

https://github.com/btpf/Alexandria/releases/tag/v0.13.1

Reply:
Sent the MSI version in https://github.com/microsoft/winget-pkgs/pull/234874